rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js

```diff
@@ -110,7 +110,8 @@ function GradebookTab(button, content) {
 
             var layout = {
                 title: greadebookTab.studentInfo[element.target.dataset['position']].username,
-                showlegend: false
+                showlegend: false,
+                xaxis: {domain: [0, 0.97]}
```

Comment:
Please add trailing comma after the last item in the container.

Reply:
done